InvenTree/order/views.py

```diff
@@ -1095,7 +1097,10 @@ def order_items(self):
                         sp=item.order_supplier))
                     continue
 
-                order.add_line_item(supplier_part, quantity)
+                # get purchase price
+                purchase_price = item.purchase_price if item.purchase_price else None
```

Comment:
Wouldn't `purchase_price` be `None` already if it is not set?

Reply:
removed that bit